src/content/docs/en/guides/cms/builderio.mdx

```diff
@@ -3,19 +3,507 @@ title: Builder.io & Astro
+const encodedUrl = encodeURIComponent("moot");
+
+const { html: postHTML } = await fetch(
+  `https://cdn.builder.io/api/v1/qwik/${builderModel}?${new URLSearchParams({
+    apiKey: builderAPIpublicKey,
+    url: encodedUrl,
+    "query.data.slug": slug,
+    cachebust: "true",
+  }).toString()}`
+)
+  .then((res) => res.json())
+  .catch();
+---
+
+<html lang="en">
+  <head>
+    <title>{title}</title>
+  </head>
+  <body>
+    <header>This is your header</header>
+
+    <article>
+      <Fragment set:html={postHTML} />
+    </article>
+
+    <footer>The is your footer</footer>
+  </body>
+</html>
+
+```
+
+This page matches [the `/builder-preview` page you created previously](#setting-up-the-preview).
+
+Now when you click on a link on your index route, you should be able to see the blog post!
+
+### Publishing your site
+
+To deploy your website, visit our [deployment guides](/en/guides/deploy/) and follow the instructions for your preferred hosting provider.
+
+#### Rebuild on Builder changes
+
+If your project is using Astro's default static mode, you will need to set up a webhook to trigger a new build when your content changes. If you are using Netlify or Vercel as your hosting provider, you can use its webhook feature to trigger a new build whenever you click **Publish** in the Builder editor.
+
+##### Netlify
+
+1. Go to your site dashboard, then **Site Settings** and click on **Build & deploy**.
+2. Under the **Continuous Deployment** tab, find the **Build hooks** section and click on **Add build hook**.
+3. Provide a name for your webhook and select the branch you want to trigger the build on. Click on **Save** and copy the generated URL.
+
+##### Vercel
+
+1. Go to your project dashboard and click on **Settings**.
+2. Under the **Git** tab, find the **Deploy Hooks** section.
+3. Provide a name for your webhook and the branch you want to trigger the build on. Click **Add** and copy the generated URL.
+
+##### Adding a webhook to Builder
+
+1. Have the URL ready that your hosting provider generated [in the previous step](#rebuild-on-builder-changes).
+2. In your Builder dashboard go to the **Models** tab, and pick the **`blogpost`** model. 
+3. In the new dialog, click on **Show More Options** and then on **Edit Webhooks** at the bottom.
+4. Now add a new webhook by clicking on **Webhook** and then expand the webhook that appeared by clicking on it.
+5. Paste the URL from step 0 that was generated by your hosting provider into the **Url** field.
+6. The last thing to do is to click on **Show Advanced** under the URL field and click the toggle so that the option **Disable Payload** is **enabled**. Click on **Done** and you're done.
+
+With this set up in place, when you click on the **Publish** button, Builder lets your hosting provider know that new content was added. This causes your site to be rebuilt and during this process Astro fetches the newly published data. Nothing to do but lean back and pump out that sweet sweet content.
```

Comment:
Comma nit
```suggestion
With this set up in place, when you click on the **Publish** button, Builder lets your hosting provider know that new content was added. This causes your site to be rebuilt and during this process, Astro fetches the newly published data. Nothing to do but lean back and pump out that sweet sweet content.
```

Reply:
Interesting, I stumble on this comma and do a double take. 

I defer the decision to y'all native English speakers. 

(I get that this is analogous to your other comma suggestions - the comma separating an adverbial phrase or clause or *frontend adverbial* from the rest of the sentence. But with this adverbial here, coming after an 'and' *and* being a really short one, it hits me differently)